packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -346,35 +437,95 @@ function WorklistDataGrid(props) {
     }
   }, [hasSelectableRows, intl, onColumnSelect]);
 
-  const handleCellSelection = useCallback((cellRowIdColId, cellCoordinates, cellSelectable) => {
+  const handleCellSelectionChange = useCallback((rowId, columnId, cellCoordinates) => {
     if (!hasSelectableRows) {
-      setAriaLiveMessage(intl.formatMessage({ id: 'Terra.worklist-data-grid.cell-selection-template' },
-        { row: cellCoordinates.row + 1, column: cellCoordinates.col + 1 }));
+      setAriaLiveMessage(intl.formatMessage({
+        id: rowId ? 'Terra.worklist-data-grid.cell-selection-template' : 'Terra.worklist-data-grid.cell-selection-cleared',
+      }, { row: cellCoordinates.row + 1, column: cellCoordinates.col + 1 }));
```

Comment:
Looks like we added back logic that we removed to simplify the implementation and make more readable.

Reply:
Removed